Documentation/BuildInstructions.md

```diff
@@ -107,7 +107,16 @@ Outside of QEMU, Serenity will run on VirtualBox. If you're curious, see how to
 
 Later on, when you `git pull` to get the latest changes, there's no need to rebuild the toolchain. You can simply run `make install`, `make image`, `make run` again. CMake will only rebuild those parts that have been updated.
 
-You may also want to replace `make` with `ninja` in the above commands for some additional build speed benefits. To do this, go to an empty directory at the root and call `cmake .. -G Ninja` inside that directory. You might either create a new directory or reuse the existing `Build` directory after cleaning it.
+#### Faster than make: "Ninja"
+
+You may also want to replace `make` with `ninja` in the above commands for some additional build speed benefits, like reduced double-building of headers.
+Most of the process stays the same:
+- Go to an empty directory at the root (e.g. `Build/`) and call `cmake .. -G Ninja` inside that directory
+- You might either create a new directory or reuse the existing `Build` directory after cleaning it.
+- `make` becomes `cmake --build .` (or possibly `cmake --build . --verbose -j 4`)
+- `make install` becomes `cmake --install .` (or possibly `cmake --install . --strip`, if you know what you're doing)
+- `make image` becomes `cmake --build . -- image`
```

Comment:
Don't all the targets exist for ninja as well? 
- ninja 
- ninja install
- ninja image
- ninja run

Reply:
Ah, for some reason I thought "ninja probably needs some extra weird arguments", looked at how `./BuildIt.sh` works, found `cmake --build . --target LibC`, and then it was all downhill from there ^^'

I'll simplify it then.

Reply:
Done.